Automattic/woocommerce-payments
# Change the copy and image on the Connect Account screen

Updated the text in the `Connect` screen so it's a bit more vague (it doesn'm make sense to tell the merchant to "verify their business" if the next step of the flow is going to be to connect Jetpack).

To test:
- Install WCPay in a clean site, or delete your `WCPay <-> Stripe` association running `wp wcpay disconnect YOUR_BLOG_ID` in a sandbox.
- You'll still need Jetpack-the-plugin installed, activated, and connected.
- Click on the `Payments` WP-Admin menu item.
- You should see the new design, with the ginormous credit card, and the button should say "Setup"..

Design mock-up:
<img width="1241" alt="Screenshot 2020-05-04 at 15 31 48" src="https://user-images.githubusercontent.com/1715800/80977735-0c503400-8e1d-11ea-8bee-0bd944d366f5.png">

Connect screen in `master`:
<img width="905" alt="Screenshot 2020-05-04 at 15 32 36" src="https://user-images.githubusercontent.com/1715800/80977743-0eb28e00-8e1d-11ea-82d2-9b441453e944.png">

Connect screen in this PR:
<img width="905" alt="Screenshot 2020-05-04 at 15 34 29" src="https://user-images.githubusercontent.com/1715800/80977745-0eb28e00-8e1d-11ea-99a1-cd612ed9a715.png">

cc/ @LevinMedia
